source/water/index.rst

```diff
@@ -1,2 +1,67 @@
 Water
 ============
+
+The water withdrawal and return flows from energy technologies are calculated in 
+MESSAGE following the approach described in Fricko et al., (2016) :cite:`fricko_2016`.
+Each technology is prescribed a water withdrawal and consumption intensity (e.g., m3 per kWh)
+that translates technology outputs optimized in MESSAGE into water requirements and return
+flows. 
+
+.. figure:: /_static/ppl_energy_balance.png
+   :width: 320px
+   :align: right
+   
+   Simplified power plant energy balance.
+   
+For power plant cooling technologies, the amount of water required and energy dissipated
+to water bodies as heat is linked to the parameterized power plant fuel conversion efficiency (heat
+rate). Looking at a simple thermal energy balance at the power plant (figure to the right), total combustion 
+energy (:math:`E_{comb}`) is conveterted into electricity (:math:`E_{elec}`), emissions (:math:`E_{emis}`)
+and additional thermal energy that must be absorbed by the cooling system (:math:`E_{cool}`):
+
+:math:`E_{comb} = E_{elec} + E_{emis} + E_{cool}`
+
+Converting to per unit electricity, we can estimate the cooling required per unit of electricity generation 
+(:math:`\phi_{cool}`) based on average heat-rate (:math:`\phi_{comb}`) and heat lost to emissions 
+(:math:`\phi_{emis}`), and this data is identified from the literature :cite:`fricko_2016`.
+
+:math:`\phi_{cool} = \phi_{comb} - \phi_{emis} - 1`
+
+With time-varying heat-rates (i.e., :math:`t =0,1,2,...`) and a constant share of energy to emissions and electricity:
+
+:math:`\phi_{cool}[t] = \phi_{comb}[t] \cdot \left( \, 1 - \dfrac{\phi_{emis}}{\phi_{comb}[0]} \, \right) - 1`
+
+Increased fuel efficiency (lower heat-rate) reduces the per unit electricity cooling requirement.   
```

Comment:
Would it be correct to write “…reduces the cooling water requirement per unit of electricity generated.” ? That would be easier to read.

Reply:
Thanks for the review. I updated the scripts in the commit below